mllib-tests/v1p4/src/main/scala/mllib/perf/clustering/LDATest.scala

```diff
@@ -0,0 +1,90 @@
+package mllib.perf.clustering
+
+import mllib.perf.PerfTest
+
+import org.json4s.JValue
+import org.json4s.JsonDSL._
+
+import scala.collection.mutable.{HashMap => MHashMap}
+
+import org.apache.commons.math3.random.Well19937c
+import org.apache.spark.SparkContext
+import org.apache.spark.mllib.clustering.LDA
+import org.apache.spark.mllib.linalg.{Vector, Vectors}
+import org.apache.spark.rdd.RDD
+
+abstract class LDATest(sc: SparkContext) extends PerfTest {
+
+  val Num_DOCUMENTS = ("num-documents", "number of documents in corpus")
```

Comment:
Which other tests (they should probably be fixed, see [Style Guide: Constants](https://github.com/databricks/scala-style-guide#naming))? FPM and GMM both use `NUM_POINTS`, `NUM_FEATURES`, etc